pkg/cloudprovider/vsphere/nsxt/config/types_ini_legacy.go

```diff
@@ -0,0 +1,35 @@
+/*
+ Copyright 2020 The Kubernetes Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package config
```

Comment:
I think it would make sense to move this to something like `pkg/nsxt/config` instead, what do you think?

Reply:
OK, make sense to me